src/Services/LDAPService.php

```diff
@@ -640,6 +656,12 @@ private function processThumbnailPhoto(Member $member, $fieldName, $data, $attri
         $existingObj = $member->getComponent($fieldName);
         if ($existingObj && $existingObj->exists()) {
             $file = $existingObj;
+
+            // If the file hashes match, and the file already exists, we don't need to update anything.
+            $hash = $existingObj->File->getHash();
+            if (hash_equals($hash, sha1($data[$attributeName]))) {
+                return;
```

Comment:
This should be `return true` to indicate success (not needing to change the file is still a success).

Reply:
resolved